packages/db/src/core/integration/vite-plugin-db.ts

```diff
@@ -114,17 +114,20 @@ export function getLocalVirtualModContents({
 		(name) => new URL(name, getDbDirectoryUrl('file:///')).pathname
 	);
 	const resolveId = (id: string) => (id.startsWith('.') ? resolve(fileURLToPath(root), id) : id);
-	const integrationSeedFilePaths = seedFiles.map((pathOrUrl) =>
-		typeof pathOrUrl === 'string' ? resolveId(pathOrUrl) : pathOrUrl.pathname
-	);
-	const integrationSeedImports = integrationSeedFilePaths.map(
-		(filePath) => `() => import(${JSON.stringify(filePath)})`
-	);
+	const integrationSeedImportStatements: string[] = [];
```

Comment:
Maybe leave a comment on why we avoided dynamic imports? Something like: "Use top-level imports to correctly resolve astro:db. Dynamic imports cause a silent build failure, potentially because of circular module references.'

Reply:
Good idea! Will add this:

```suggestion
	// Create static imports block to correctly resolve `astro:db` in integration seed files.
	// Dynamic imports cause a silent build failure, potentially because of circular module references.
	const integrationSeedImportStatements: string[] = [];
```

Reply:
Oop, see you’ve got it! Go Ben 🙌